simulation/policies.py

```diff
@@ -89,13 +94,15 @@ def su_add_to_network(params, step, sL, s, _input, **kwargs):
         network = s["network"]
         funding_pool = s["funding_pool"]
         token_supply = s["token_supply"]
+        gamma = params["gamma_func"]
+        random_number = params["random_number_func"]
```

Comment:
For the random functions, please leave the "_func" in the name until the very end when we get a number out of it.
With 'probability' it was okay because you think of probability as "something that has to be calculated", but now that we have many random functions and values, stricter syntax to distinguish between when it is still a function, and when it has become a value becomes more useful.